examples/basic/server.js

```diff
@@ -69,13 +59,28 @@ await app.register(fastifyStatic, {
   lastModified: true,
 });
 
-app.all("*", async (request, reply) => {
-  if (process.env.NODE_ENV === "production") {
-    let links = getEarlyHintLinks(request, initialBuild);
-    await reply.writeEarlyHintsLinks(links);
-  }
+app.register(async function (childServer) {
+  childServer.removeAllContentTypeParsers();
```

Comment:
remove all app level content parsers so we can allow all requests bodies to flow through to remix